.github/workflows/cherryPick.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         with:
           ref: staging
           token: ${{ secrets.OS_BOTIFY_TOKEN }}
+          submodules: true
```

Comment:
This ensures that we can CP the change in the Mobile-Expensify version hash to staging. Without it, Mobile-Expensify submodule will be missing and the cherry-pick can't change what's not there